src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -2444,8 +2445,10 @@ namespace ReportSizingManager {
                     } else {
                         AutosizeDes = FinalSysSizing(CurSysNum).CoolSupHumRat;
                     }
-                    if (AutosizeDes > DataDesInletAirHumRat && DataCapacityUsedForSizing > 0.0 &&
-                        UtilityRoutines::SameString(CompType, "COIL:COOLING:WATER")) { // flow here is water vol flow rate
+                    if (AutosizeDes > DataDesInletAirHumRat &&
+                        UtilityRoutines::SameString(CompType,
+                                                    "COIL:COOLING:WATER")) { // flow here is water vol flow rate, take out DataCapacityUsedForSizing >
```

Comment:
Shouldn't these warnings also apply to the detailed cooling coil model?

Reply:
Or any cooling coil for that matter.

Reply:
Well, only water coils have an inlet water temp so OK, simple and detailed then.